### DISCUSSION TITLE
part-ii/2016-0/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#180] [Question comments]
Part II 2016 Algebraic Geometry: Paper 3, Section II, H 

https://questions.tripos.org/part-ii/2016-0/

**Comment by silkca** [comment]
The first part is all bookwork.

The second part is a simple calculation - the thing that's gone wrong can be investigated further by considering the tangencies of $Z(y),Z(y^2-x^3+xy^3) \subseteq \mathbb{A}^{2}$ at their intersection.

The final part is just a standard calculation from the definitions.
